javascript/exorelay/src/websocket-connector/websocket-connector.ls

```diff
@@ -60,6 +61,12 @@ class WebSocketConnector extends EventEmitter
       ..on 'open', @_on-socket-open
       ..on 'message', @_on-socket-message
       ..on 'error', @_on-socket-error
+      ..on 'close', @_on-socket-close
+
+
+  _on-socket-close: ~>
+    unless @socket-killed
+      @connect
```

Comment:
shouldn't you be calling connect? Please investigate why the test is passing before fixing this.